tests/www/presenters/test_show_my_cooperations_presenter.py

```diff
@@ -153,19 +153,14 @@ def test_successfull_accept_request_response_is_presented_correctly(self):
             None,
             get_coop_plans_response_length_1(),
         )
-        self.assertTrue(presentation_success.accept_message_success)
-        self.assertFalse(presentation_success.deny_message_success)
-        self.assertEqual(
-            len(presentation_success.accept_message),
-            1,
-        )
-        self.assertEqual(
-            presentation_success.accept_message[0],
-            self.translator.gettext("Cooperation request has been accepted."),
+        assert len(self.notifier.infos) == 1
+        assert self.notifier.infos[0] == self.translator.gettext(
+            "Cooperation request has been accepted."
         )
+        assert not self.notifier.warnings
```

Comment:
Just a pedantic point, if the warning notification array is checked to be empty/None here shouldn't it be checked in the other tests (but the other way around possibly depending on the case)? Since I suppose it's just as likely in the other cases the other notification array gets inadvertently filled.